site/docs/guides/operations/kubernetes/healthchecks.md

```diff
@@ -0,0 +1,257 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project 2020. -->
+
+# Healthchecks when running Egeria in Kubernetes
+
+There are various API calls that will check the status of Egeria.
+
+These may be typically used in a Kubernetes environment to check if Egeria is ready to service requests. Here we summarize what is available.
+
+## Defining a Kubernetes health check
+
+See also [Kubernetes docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
+
+Kubernetes has 3 types of health checks
+- startup -- to confirm a pod has started
+- readiness -- to confirm a pod is ready. Typically this will then allow requests to be routed here
+- liveness -- to check the pod is still responding to requests in a timely fashion.
+
+Typically pods will be restarted if these health checks do not pass in a specified time period.
+
+Each of these checks can be of several types
+- tcpSocket -- this just checks for an open port.
+- grpc -- issues grpc call (we do no use grpc in egeria)
+- httpGet -- a simple GET. If return is >=200 and <400 it is successful
+- exec -- issues a specified command within the container
+- 
+## Example Egeria API calls
+
+In these examples the [httpie](https://httpie.io) tool will be used as it will print both the response code, and pretty-formatted body by default. Other tools like curl may also be used, but more parsing may be required of the responses.
+
+The examples here were run against the lab charts, using user 'garygeeke'. A simple security plugin is active which restricts user access to api calls.
+
+### Platform
+This checks if the *platform* is available.
+
+#### Platform is not running
+➜  ~ curl -k -X GET --connect-timeout 5 --max-time 5 "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/server-origin"
+Egeria OMAG Server Platform (version 4.1-SNAPSHOT)
+
+#### Platform is running
+➜  ~ http --verify=no --pretty=format GET "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/server-origin"
+HTTP/1.1 200
+Connection: keep-alive
+Content-Length: 42
+Content-Type: text/plain;charset=UTF-8
+Date: Thu, 18 May 2023 17:10:27 GMT
+Keep-Alive: timeout=60
+
+Egeria OMAG Server Platform (version 4.0)
+
+
+
+### Server
+This API call was introduced in [release 3.4](/release-notes/3-4.md)
+
+#### Server is not known
+```
```

Comment:
This call indicates whether the server is running or not - the call to determine whether the server is known on the platform is `{baseURL}/open-metadata/platform-services/users/{user}/server-platform/servers/{serverName}/is-known`